recipe-server/normandy/base/views.py

```diff
@@ -1,5 +1,9 @@
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseRedirect
 
 
 def index(request):
     return HttpResponse('<h1>Normandy</h1>')
+
+
+def favicon(request):
+    return HttpResponseRedirect('/static/img/favicon.ico')
```

Comment:
Hardcoding this url means that it won't work on our production servers, which serve static content with a CDN. You're better off building this url with [django.contrib.staticfiles.templatetags.staticfiles.static](https://stackoverflow.com/questions/11721818/django-get-the-static-files-url-in-view/17738606#17738606), which will work in all environments.